static/app/views/alerts/list/incidents/row.tsx

```diff
@@ -99,10 +100,8 @@ const ProjectBadge = styled(IdBadge)`
   flex-shrink: 0;
 `;
 
-const FlexCenter = styled('div')`
+const FlexCenter = styled(Flex)`
```

Comment:
this doesn't work

Reply:
Unfortunately, I don't think this one works either. I've reverted the change and wrote a story to test (using PanelTable as the wrapper to mimic the usage), and this is what I see

![CleanShot 2025-06-18 at 14 28 29@2x](https://github.com/user-attachments/assets/9e6eab18-9518-4672-ad75-3854d596c36b)